Source/WebKit/WebProcess/WebProcess.h

```diff
@@ -703,7 +703,7 @@ class WebProcess : public AuxiliaryProcess
 
     HashMap<WebCore::FrameIdentifier, WeakPtr<WebFrame>> m_frameMap;
 
-    using WebProcessSupplementMap = HashMap<ASCIILiteral, std::unique_ptr<WebProcessSupplement>, ASCIILiteralPtrHash>;
```

Comment:
But why are we still not dropping the ASCIILiteralPtrHash implementation? Is there still some use?

Reply:
We can once the other PR and this one lands.